Sources/SwiftGenKit/Parsers/JSON/JSONFile.swift

```diff
@@ -0,0 +1,32 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+public extension JSON {
+  struct File {
+    let path: Path
+    let name: String
+    let documents: [Any]
```

Comment:
Maybe we could already change that to `let content: Any` (and replace `self.documents = [item]` by `self.content = item` in `init`), and to the "wrapping in a `documents` array" (to keep API compatibility) only in `stencilContext()` (i.e. replace `"documents": file.documents.map(map(document:))` in the exposed dict with `"documents": map(content: file.content)`)?

Reply:
that's a good idea!